src/containers/AdminScriptImport.jsx

```diff
@@ -20,19 +20,26 @@ const styles = StyleSheet.create({
 export default class AdminScriptImport extends Component {
   static propTypes = {
     startImport: PropTypes.func,
-    hasPendingJob: PropTypes.bool
+    hasPendingJob: PropTypes.bool,
+    jobError: PropTypes.bool,
+    onSubmit: PropTypes.bool
   };
 
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      ...(!!props.jobError && {
+        error: `Error from last attempt: ${props.jobError}`
+      })
+    };
   }
 
   startImport = async () => {
     const res = await this.props.startImport(this.state.url);
     if (res.errors) {
       this.setState({ error: res.errors.message });
     }
+    this.props.onSubmit();
```

Comment:
is this on submit necessary?

Reply:
I think so. It causes the section to close.

Reply:
Ok, this component is kind of weird. It seems like startImport is the "real" onSubmit and onSubmit here is just being called to advance sections. If you call onSubmit you can get rid of the redundant refetchQueries on the import script mutation.